sc-models/knowledge-base/faq.gwf

```diff
@@ -0,0 +1,865 @@
+        <node type="node/const/perm/relation" idtf="nrel_answer" shapeColor="0" id="94712554872912" parent="0" left="0" top="0" right="98.3906" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="326.5" y="2948.5" haveBus="false" idtf_pos="4">
+            <content type="0" mime_type="" content_visibility="false" file_name=""/>
+        </node>
+        <node type="node/const/perm/general" idtf="question_metasystem_ims" shapeColor="0" id="94712564920080" parent="0" left="0" top="0" right="200.438" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="148" y="3020" haveBus="false" idtf_pos="1">
+            <content type="1" mime_type="content/term" content_visibility="true" file_name=""><![CDATA[Где можно познакомиться с Метасистемой OSTIS?]]></content>
+        </node>
+        <node type="node/const/perm/relation" idtf="nrel_format" shapeColor="0" id="94712570687408" parent="0" left="0" top="0" right="93.8906" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="925.5" y="2818.5" haveBus="false" idtf_pos="0">
+            <content type="0" mime_type="" content_visibility="false" file_name=""/>
+        </node>
+        <node type="node/const/perm/general" idtf="format_html" shapeColor="0" id="94712568393264" parent="0" left="0" top="0" right="98.4063" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="987.5" y="2765.5" haveBus="false" idtf_pos="0">
+            <content type="0" mime_type="" content_visibility="false" file_name=""/>
+        </node>
+        <node type="node/const/perm/general" idtf="" shapeColor="0" id="94712570013808" parent="0" x="590" y="2772" haveBus="false" idtf_pos="0">
+            <content type="0" mime_type="" content_visibility="false" file_name=""/>
+        </node>
+        <node type="node/const/perm/general" idtf="format_html" shapeColor="0" id="94712565164944" parent="0" left="0" top="0" right="98.4063" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="793.5" y="3017.5" haveBus="false" idtf_pos="0">
+            <content type="0" mime_type="" content_visibility="false" file_name=""/>
+        </node>
+        <node type="node/const/perm/general" idtf="section_faq_ostis_metasystem" shapeColor="0" id="94712565976016" parent="0" left="0" top="0" right="229.125" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="96.5" y="2911.5" haveBus="false" idtf_pos="4">
+            <content type="0" mime_type="" content_visibility="false" file_name=""/>
+        </node>
+        <node type="node/const/perm/group" idtf="question" shapeColor="0" id="94712570473888" parent="0" left="0" top="0" right="71.9219" bottom="25" textColor="164" text_angle="0" text_font="Times New Roman [Arial]" font_size="10" x="134.5" y="2885.5" haveBus="false" idtf_pos="2">
```

Comment:
I think, "question" is not correct class for all the links. Question is `action` but deprecated ^). Just remove this class from all the questions, it would be OK